android/java/org/chromium/chrome/browser/crypto_wallet/util/AndroidUtils.java

```diff
@@ -43,4 +47,37 @@ public static Spanned formatHTML(String html) {
             return Html.fromHtml(html);
         }
     }
+
+    // Views
+    public static TextView makeHeaderTv(Context context) {
+        TextView textView = new TextView(context);
+        textView.setTextAppearance(R.style.BraveWalletTextViewTitle);
+        textView.setText(R.string.brave_wallet_solana_max_retries);
+        textView.setTypeface(null, Typeface.BOLD);
+        textView.setId(View.generateViewId());
+        return textView;
+    }
+
+    public static TextView makeSubHeaderTv(Context context) {
```

Comment:
the same as above regarding a function name.

Reply:
Same as above, just a normal textview to be used as a sub-header with no additional checks etc to avoid extra responsibilities or checks. Can add more helper later if required.

Reply:
same question as above, why `textView.setText(R.string.brave_wallet_solana_max_retries);` and not `textView.setText(R.string.ok);`?

Reply:
as above, missed to remove that, will remove, thanks!